notebooks/COS/AsnFile/AsnFile.ipynb

```diff
@@ -6,8 +6,8 @@
    "source": [
```

Comment:
<p>Hi Sierra, thank you for initiating the notebook review process.</p><p>I see the following links are not correctly linked within the notebook itself:</p><ul><li> <a href="https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/32/#2-editing-an-existing-association-file" target="_blank">editing an existing&nbsp;</a><code><a href="https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/32/#2-editing-an-existing-association-file" target="_blank">asn</a></code><a href="https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/32/#2-editing-an-existing-association-file" target="_blank">&nbsp;file to add or remove an exposure</a>.</li><li><a href="https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/32/#332-example-for-split-wavecal-dataAF" target="_blank">creating an&nbsp;</a><code><a href="https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/32/#332-example-for-split-wavecal-dataAF" target="_blank">asn</a></code><a href="https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/32/#332-example-for-split-wavecal-dataAF" target="_blank">&nbsp;file for use with SPLIT wavecal data</a> </li></ul><p>I see their target has an unmatched #id in the tag.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>